modin/core/io/column_stores/column_store_dispatcher.py

```diff
@@ -164,9 +166,11 @@ def build_columns(cls, columns):
             if len(columns) % num_partitions == 0
             else len(columns) // num_partitions + 1
         )
+        # If columns is empty, column_splits == 0, but the third argument, the
+        # step, in the range constructor must be nonzero, so use 1 instead.
         col_partitions = [
             columns[i : i + column_splits]
-            for i in range(0, len(columns), column_splits)
+            for i in range(0, len(columns), max(column_splits, 1))
```

Comment:
If we are reading an empty parquet file, `len(columns)` and `column_splits` should both be 0, right? Maybe in that case, it makes sense to have a short circuiting if statement that just creates an empty list like so?
```python
if len(columns) == 0:
    col_partitions = [[]]
else:
...
```

Reply:
Done.